doc/release/release_0.14.rst

```diff
@@ -0,0 +1,297 @@
+- Fixed ``skimage.measure.find_contours`` raising StopIteration on Python 3.7. (#3038)
+- Fixed platform-specific issues appearing in Windows and/or 32-bit environments. (#2867, #3033)
+
+
+API Changes
+-----------
+- ``skimage.util.montage.`` namespace has been removed, and
+  ``skimage.util.montage.montage2d`` function is now available as
+  ``skimage.util.montage2d``.
+- ``skimage.morphology.binary_erosion`` now uses ``True`` as border
+  value, and is now consistent with ``skimage.morphology.erosion``.
+
+
+Deprecations
+------------
+- ``skimage.util.montage2d`` is deprecated and will be removed in 0.15.
+  Use ``skimage.util.montage`` function instead.
+- ``skimage.novice`` is deprecated and will be removed in 0.16.
+- ``skimage.transform.resize`` and ``skimage.transform.rescale`` have a new
+  ``anti_aliasing`` option that avoids aliasing artifacts when down-sampling
+  images. This option will be enabled by default in 0.15.
+- ``regionprops`` will use row-column coordinates in 0.16. You can start
+  using them now with ``regionprops(..., coordinates='rc')``. You can silence
+  warning messages, and retain the old behavior, with
+  ``regionprops(..., coordinates='xy')``. However, that option will go away
+  in 0.16 and result in an error. This change has a number of consequences.
+  Specifically, the "orientation" region property will measure the
+  anticlockwise angle from a *vertical* line, i.e. from the vector (1, 0) in
+  row-column coordinates.
+- ``skimage.morphology.remove_small_holes`` ``min_size`` argument is deprecated
+  and will be removed in 0.16. Use ``area_threshold`` instead.
+
+
+Contributors to this release
+----------------------------
+
+- Alvin
+- Norman Barker
+- Leonid Bloch
+- Benedikt Boecking
+- François Boulogne
+- Larry Bradley
+- Matthew Brett
+- Alex Chum
+- Yannick Copin
+- Nethanel Elzas
+- Kira Evans
+- Christoph Gohlke
+- Peter Goldsborough
+- Emmanuelle Gouillart
+- Ben Hadfield
+- Mark Harfouche
+- Scott Heatwole
+- Gregory R. Lee
+- Guillaume Lemaitre
+- Kevin Mader
+- Jarrod Millman
+- Pradyumna Narayana
+- Juan Nunez-Iglesias
+- Egor Panfilov
+- Oleksandr Pavlyk
+- Alex Rothberg
+- Max Schambach
+- Johannes Schönberger
+- Matt Swain
+- Saurav R. Tuladhar
+- Nelle Varoquaux
+- Viraj
+- David Volgyes
+- Stefan van der Walt
+- Thomas Walter
+- Scott Warchal
+- Nicholas Weir
+- corrado9999
+- ed1d1a8d
+- eepaillard
+- mikigom
+- mutterer
+
+
+We'd also like to thank all the people who contributed their time to perform the reviews:
+
+- Leonid Bloch
+- Jirka Borovec
+- François Boulogne
+- Kira Evans
+- Peter Goldsborough
+- Emmanuelle Gouillart
+- Almar Klein
+- Gregory R. Lee
+- Joan Massich
+- Juan Nunez-Iglesias
+- Daniil Pakhomov
+- Egor Panfilov
+- Johannes Schönberger
+- Steven Silvester
+- Stefan van der Walt
+- Josh Warner
+- Eric Wieser
+
+
+Full list of changes
+--------------------
+This release is the result of 8 months of work.
```

Comment:
8 months -> 12 or 14?  I haven't checked the exact dates.

Reply:
@soupault yes, there is work in 0.14 that didn't go into 0.13.1

Reply:
In fact, 0.13.1 only has a few backported PRs from the time between 0.13.0 and 0.13.1. But most PRs did not go in there.

Reply:
@jni Oh, it's been a while. I'm taking a look into this.

Reply:
Done for this point.